conf/feleConf.json

```diff
@@ -1,7 +1,7 @@
 {
     "couchdb": {
         "username" : "admin",
-        "password" : "u1234",
+        "password" : "icui4cu",
         "port": 5984
     },
     "logging": {
```

Comment:
Should we include this file in `.gitignore` or is the username and password the same for everyone ?

Reply:
One thing we can do is we can add a .env file and put everything in that

Reply:
Since we are running couchdb locally, everyone has different credentials.  I forgot to change them. It's better to use local environment

Reply:
Yeah adding a `.env` sounds like a better idea